sources/academy/webscraping/anti_scraping/index.md

```diff
@@ -93,7 +93,7 @@ A common workflow of a website after it has detected a bot goes as follows:
 
 One thing to keep in mind while navigating through this course is that advanced scraping methods are able to identify non-humans not only by one value (such as a single header value, or IP address), but are able to identify them through more complex things such as header combinations.
 
-A conference talk by [Ondra Urban](https://github.com/mnmkng) will guide you through various anti-scraping measures and how to get around them.
+Watch a conference talk by [Ondra Urban](https://github.com/mnmkng), which provides an overview of various anti-scraping measures and tactics for circumventing them. Although the talk, given in 2021, features some outdated code examples, it still serves well as a general overview.
```

Comment:
maybe we could accentuate that the talk is outdated but still worth it by utilizing some admonition?

Reply:
Addressed in 3eddcda12399665167d2969ee956ca21cc596efd

Reply:
A bit more whitespace would help, but I don't feel like digging into CSS of the docs right now 😬 

<img width="779" alt="Screenshot 2024-05-21 at 14 41 39" src="https://github.com/apify/apify-docs/assets/283441/c8b727d5-9222-4152-ae99-81f409a52ddb">